modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/portlet-model-hints.xml

```diff
@@ -72,7 +72,7 @@
 			<validator name="required" />
 		</field>
 		<field localized="true" name="description" type="String">
-			<hint-collection name="TEXTAREA" />
+			<hint-collection name="CLOB" />
```

Comment:
Hi @rodrigopaulino, why did you decide to change the column's data type to CLOB? According to our discussion on [PTR-1777](https://issues.liferay.com/browse/PTR-1777), I remember that you decided to not do this. Can you please let me know why did you change your mind?

Thanks.

Reply:
Sure thing, @natocesarrego. Including a validation to limit the description field length is much more difficult than expected. Since we can't predict how much characters a translation is going to take, how many languages the user is going to use and if they plan to add more to the set of available languages in the future, there's no way to determine a significant size limit that can cover all cases. Also, changing the column to CLOB is, in fact, a micro change. You can check the full discussion about this decision with the Upgrade SME, Alberto Chaparro, in the [PTR-1804](https://issues.liferay.com/browse/PTR-1804).